docs/capabilities/prepare-tools.md

```
@@ -0,0 +1,33 @@
+# Prepare Tools
+
+[`PrepareTools`][pydantic_ai.capabilities.PrepareTools] and [`PrepareOutputTools`][pydantic_ai.capabilities.PrepareOutputTools] wrap a [`ToolsPrepareFunc`][pydantic_ai.tools.ToolsPrepareFunc] as a [capability](overview.md), for filtering or modifying [tool definitions](../tools.md) per step. `PrepareTools` handles function tools; `PrepareOutputTools` handles [output tools][pydantic_ai.output.ToolOutput].
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "pydanticai.output.tooloutput"

(MD052, reference-links-images)